src/commands/junit/README.md

```diff
@@ -40,6 +40,7 @@ Additionally you might configure the `junit` command with environment variables:
 - `DD_TAGS`: set global tags applied to all spans. The format must be `key1:value1,key2:value2`.
   - The resulting dictionary will be merged with whatever is in the `--tags` parameter. If a `key` appears both in `--tags` and `DD_TAGS`, whatever value is in `DD_TAGS` will take precedence.
 - `DATADOG_SITE`: choose your Datadog site, e.g. datadoghq.com or datadoghq.eu.
+- `DATADOG_JUNIT_INTAKE_URL`: override the full URL for the intake endpoint.
```

Comment:
is this something we want to show to users? Are we doing it for sourcemaps?

Reply:
it seems more like an internal thing

Reply:
FYI in the sourcemaps readme we do have it mentioned [here](https://github.com/DataDog/datadog-ci/tree/master/src/commands/sourcemaps#setup):  
````
It is also possible to override the full URL for the intake endpoint by defining the DATADOG_SOURCEMAP_INTAKE_URL environment variable.
```

Reply:
oh sry I missed that. In any case, is there any reason to show it to users?

Reply:
I guess the only reason would be for the proxy case so customers can easily know what to configure.